NEWS.rst

```diff
@@ -7,6 +7,15 @@
 
 .. towncrier release notes start
 
+19.0.4 (2019-03-04)
+===================
+
+Bug Fixes
+---------
+
+- add a warning when tar file is placed outside of target location. (`#3907 <https://github.com/pypa/pip/issues/3907>`_)
```

Comment:
Please undo the changes in this file. This is auto-generated at release time using the NEWS fragments added in `news/` directory.

Reply:
changes has been undo